### ISSUE TITLE
Progress bar for the most time-consuming tasks?

### ISSUE THREAD

**Issue by diogoprov** [naturalis/supersmart#64]
Just a detail for now, but maybe for v2.0, it'd be nice to have a progress bar sth like this (http://stackoverflow.com/questions/238073/how-to-add-a-progress-bar-to-a-shell-script) for some of the most time consuming functions, like smrt align.
Just a thought

**Comment by rvosa** [comment]
Yup, we've already been thinking about how best to do that. Would definitely be nice to have.
